client/gutenberg/extensions/map/add-point/index.js

```diff
@@ -0,0 +1,45 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
```

Comment:
To get strings translated for both Jetpack and Calypso, we're using a `@wordpress/i18n` wrapper for now ():

```js
import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
```

In Translation functions you could then drop out the jetpack textdomain:
`_( 'Yo!', 'jetpack' )` → `_( 'Yo!' )`

More info:
https://github.com/Automattic/wp-calypso/blob/d51de00c263eaeca36b7e5918bfba72bddea20d4/client/gutenberg/extensions/presets/jetpack/utils/i18n.js

Reply:
Removed the textdomain everywhere: https://github.com/Automattic/wp-calypso/pull/28498/commits/eaebff071c15c933269cff7a1969db9c7be33017